lms/services.py

```diff
@@ -0,0 +1,30 @@
+import stripe
+
+from config.settings import api_key
```

Comment:
Импорт нужно сделать из django (иначе потом не сможешь тестировать)
```suggestion
from django.conf import settings
```
И ключ занести в stripe только в одном месте (в глобальной области)
```python
stripe.api_key = settings.STRIPE_API_KEY